docs/configuration/cloudwatch-retention.md

```diff
@@ -35,44 +35,56 @@ Changing the log retention policy in the AWS Management Console is a fairly simp
 
 ## Terraform
 
-The `cumulus` module exposes values for configuration of log retention for
-cloudwatch log groups (in days). A configurable map of `cloudwatch_log_retention_periods` currently supports the following variables:
-
-- cumulus-tf_egress_lambda_log_retention
-- archive_private_api_log_retention
-- archive_api_log_retention
-- archive_async_operation_log_retention
-- archive_granule_files_cache_updater_log_retention
-- archive_publish_executions_log_retention
-- archive_publish_granule_log_retention
-- archive_publish_pdrs_log_retention
-- archive_replay_sqs_messages_log_retention
-- cumulus_distribution_api_log_retention
-- cumulus_ecs_service_default_log_retention
-- ingest_discover_pdrs_task_log_retention
-- ingest_hyrax_metadata_updates_task_log_retention
-- ingest_parse_pdr_task_log_retention
-- ingest_post_to_cmr_task_log_retention
-- ingest_queue_pdrs_task_log_retention
-- ingest_queue_workflow_task_log_retention
-- ingest_sync_granule_task_log_retention
-- ingest_update_cmr_access_constraints_task_log_retention
-
-In order to configure this value for the cloudwatch log group, the variable for the retention period for the respective group should be in the form of:
-
-```hcl
-<cumulus_module>_<cloudwatch_log_group>_log_retention: <log_retention>
-  type = number
+There are optional variables that can be set during deployment of cumulus modules to configure
+the retention period (in days) of cloudwatch log groups for lambdas and tasks which the Cumulus module supports:
+
+```tf
+module "cumulus" {
+  # ... other variables
+  default_log_retention_days = var.default_log_retention_days
+  cloudwatch_log_retention_periods = var.cloudwatch_log_retention_periods
+}
 ```
```

Comment:
Add the other cumulus modules user would use, like "cumulus_distribution"
and "cumulus_ecs_service"